vpdsl/plugins/org.polarsys.capella.cdoxml.ta/src/org/polarsys/capella/cdoxml/ta/CapellaCDOandXMLTargetApplication.java

```diff
@@ -12,18 +12,18 @@
 package org.polarsys.capella.cdoxml.ta;
 
 import org.polarsys.capella.cdoxml.ta.genchain.CapellaConnectorExtension;
+import org.polarsys.capella.extension.genchain.CapellaCdoGenerationExtension;
 import org.polarsys.capella.ta.definition.CapellaTargetApplication;
 
 import java.util.List;
 import org.eclipse.egf.portfolio.genchain.extension.ExtensionHelper;
-import org.eclipse.egf.portfolio.genchain.cdo.CdoGenerationExtension;
 
 public class CapellaCDOandXMLTargetApplication extends CapellaTargetApplication  {
```

Comment:
I think it is better to change the super class to CDOForCapellaTargetApplication and remove the line 26.

Reply:
You are right